### PR TITLE
Add rezension link to footer

### LINKED ISSUE #68: Trust Elements
Google Bewertungen einbauen. Recherche ob infos direkt von google oder wir bauen uns ein eigenes 😈 

Link zum BewertungsFormular:
https://www.google.com/search?client=firefox-b-d&q=nordlichter+halsenbek#lrd=0x47b18128d7d26c01:0x975243cc198cbb69,1,,,

### PR DESCRIPTION
**Pull request by mpagels**
Dieser pull request fügt einen Link in den Footer ein, der zur Google Rezension für die nordlichter führt.
fixes #68

### CODE REVIEW

File: components/Footer/Footer.js [hunk 1]
```diff
@@ -3,12 +3,24 @@ import styled from 'styled-components'
...
   return (
     <Wrapper>
+      <GoogleRezession>
+        <p>
+          Waren Sie mit unser Leistung zufrieden oder haben Sie Vorschläge, wie
+          wir uns verbessern können? Dann klicken Sie auf das <a style={{"color" : "white"}} href="https://www.google.com/search?q=nordlichter+halsenbek#lrd=0x47b18128d7d26c01:0x975243cc198cbb69,1,,," target="_blank">Google Logo</a> und
+          schreiben Sie uns gerne eine Rezension. Wir freuen uns über
+          konstruktive Kritik und positive Rückmeldungen. Herzlichen Dank!
+        </p>
+        <a href="https://www.google.com/search?q=nordlichter+halsenbek#lrd=0x47b18128d7d26c01:0x975243cc198cbb69,1,,," target="_blank">
```
Comment (soerenstein) [inline components/Footer/Footer.js:20]: Same here.

```suggestion
        <a href="https://www.google.com/search?q=nordlichter+halsenbek#lrd=0x47b18128d7d26c01:0x975243cc198cbb69,1,,," target="_blank" rel="noopener noreferrer">
```

File: components/Footer/Footer.js [hunk 2]
```diff
@@ -3,12 +3,24 @@ import styled from 'styled-components'
...
 import { useMediaQuery } from 'react-responsive'
 
 export default function Footer() {
   const isDesktop = useMediaQuery({ query: '(max-width: 841px)' })
   return (
     <Wrapper>
+      <GoogleRezession>
+        <p>
+          Waren Sie mit unser Leistung zufrieden oder haben Sie Vorschläge, wie
+          wir uns verbessern können? Dann klicken Sie auf das <a style={{"color" : "white"}} href="https://www.google.com/search?q=nordlichter+halsenbek#lrd=0x47b18128d7d26c01:0x975243cc198cbb69,1,,," target="_blank">Google Logo</a> und
```
Comment (soerenstein) [inline components/Footer/Footer.js:16]: To increase security when linking to another host, I would use the rel attribute.

```suggestion
          wir uns verbessern können? Dann klicken Sie auf das <a style={{"color" : "white"}} href="https://www.google.com/search?q=nordlichter+halsenbek#lrd=0x47b18128d7d26c01:0x975243cc198cbb69,1,,," target="_blank" rel="noopener noreferrer">Google Logo</a> und
```

File: components/Footer/Footer.js [hunk 3]
```diff
@@ -3,12 +3,24 @@ import styled from 'styled-components'
...
 import Map from './Map'
 import Link from 'next/link'
+import GoogleLogoPNG from '../../assets/png/footer/google-logo.png'
 import { useMediaQuery } from 'react-responsive'
 
 export default function Footer() {
   const isDesktop = useMediaQuery({ query: '(max-width: 841px)' })
   return (
     <Wrapper>
+      <GoogleRezession>
```
Comment (soerenstein) [inline components/Footer/Footer.js:13]: Just a little typo. 

```suggestion
      <GoogleRezension>
```

File: components/Footer/Footer.js [hunk 4]
```diff
@@ -3,12 +3,24 @@ import styled from 'styled-components'
...
+        <p>
+          Waren Sie mit unser Leistung zufrieden oder haben Sie Vorschläge, wie
+          wir uns verbessern können? Dann klicken Sie auf das <a style={{"color" : "white"}} href="https://www.google.com/search?q=nordlichter+halsenbek#lrd=0x47b18128d7d26c01:0x975243cc198cbb69,1,,," target="_blank">Google Logo</a> und
+          schreiben Sie uns gerne eine Rezension. Wir freuen uns über
+          konstruktive Kritik und positive Rückmeldungen. Herzlichen Dank!
+        </p>
+        <a href="https://www.google.com/search?q=nordlichter+halsenbek#lrd=0x47b18128d7d26c01:0x975243cc198cbb69,1,,," target="_blank">
+        <GoogleLogo src={GoogleLogoPNG} alt="google-logo" /> </a>
+      </GoogleRezession>
```
Comment (soerenstein) [inline components/Footer/Footer.js:22]:
```suggestion
      </GoogleRezension>
```

**Review by soerenstein — COMMENTED** [review]
Only small suggestions, nothing necessary.

File: components/Footer/Footer.js (hunk 2 above)
Comment (mpagels) [reply components/Footer/Footer.js:16]: Das wusste ich nicht. Danke dir.

File: components/Footer/Footer.js (hunk 3 above)
Comment (mpagels) [reply components/Footer/Footer.js:13]: It's not a typo if the style component is named as this ;-) But yes, will change this. Thx

Comment (soerenstein) [reply components/Footer/Footer.js]: Oh, I didn't see the style component 😅

**Pull request merged**